build.gradle.kts

```diff
@@ -61,10 +61,6 @@ java {
   toolchain.languageVersion.set(JavaLanguageVersion.of(21))
 }
 
-kotlin {
-  jvmToolchain(21)
-}
-
```

Comment:
removed this since not there on other projects https://github.com/ministryofjustice/visit-scheduler/blob/main/build.gradle.kts and might be a reason for the security job on CircleCI failing - https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-prison-visit-booker-registry/223/workflows/abe03ec8-559e-4903-92cd-4c3c83cdf9b0/jobs/587